bullet_train-outgoing_webhooks/test/setup/active_record.rb

```diff
@@ -1,36 +1,8 @@
-require "active_record"
-
-ActiveRecord::Base.establish_connection(adapter: "sqlite3", database: ":memory:")
-
 # Set so Active Record casts created_at/updated_at to ActiveSupport::TimeWithZone.
-# (Mathces Rails apps' default, but its assigned via Active Record's railtie.)
+# (Matches Rails apps' default, but its assigned via Active Record's railtie.)
 Time.zone = "UTC"
 ActiveRecord::Base.time_zone_aware_attributes = true
 
-ActiveRecord::Schema.define do
-  create_table :teams, force: true do |t|
-    t.string :name, null: false
-    t.timestamps null: false
-  end
-
-  create_table :webhooks_outgoing_endpoints, force: true do |t|
-    t.references :team
-    t.string :url, null: false
-    t.string :name, null: false
-    t.integer :event_type_ids, array: true, default: []
```

Comment:
`sqlite3` doesn't seem to support `array: true` in CI, so I just changed this gem to test against postgres which is what we target for deployment anyway.